api/api/accounts/test_list_userroles.py

```diff
@@ -0,0 +1,58 @@
+from datetime import date
+
+import pytest
+from django.contrib.auth import get_user_model
+from rest_framework import status
+from rest_framework.test import APITestCase
+
+from api.clubs.models import Club
+from api.roles.models import Roles
```

Comment:
We should update the `Roles` class to `Role`, the class represent definition of a single object. Maybe not in this PR.